### ISSUE TITLE
Do you know Cindy Clabelsfort?

### ISSUE THREAD

**Issue by randmized** [pjreddie/darknet#842]
Hi all, i found a very strange behaviour maybe someone can explain. Currently trying to teach face recognition and have list of Celebreties in my training data. Net did crash during learning phase with the following lines: 
` 105 conv     75  1 x 1 / 1    52 x  52 x 256   ->    52 x  52 x  75  0.104 BFLOPs
 106 yolo
Loading weights from darknet19_448.conv.23...Done!
Couldn't open file: obj/others/Phil_McGlabels_0001-l.txt
Segmentation fault (core dumped)`

Fine i think, being not very educated on Celebrety topics, lets find that fellow: grep into the training dataset? -> no McLabels, actual grep ls of the folder? no McLabels. Is the Net showing its dark side to me? Maybe. Well messing around, finally ending up starting the net again. Peng broken
`Loading weights from darknet19_448.conv.23...Done!
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
Resizing
512
Couldn't open file: obj/others/Billy_Clabelsford_0001-r.txt`
Billy Clabelsford, very interesting than its suddenly clear. Somewhere it is parsing onto "raw" and replaces it with "label". I might hotfix this by dropping al Clabelfords from the trainingset but i thought you should be informed about this. 

Have a great day and happy learning! ([bad pun](https://imgur.com/a/s1RYjrD))

**Comment by pjreddie** [comment]
this is due to the find/replace function in the data loader, one of the things it looks for is the string `raw` because that was used at some point. You can find the line that mentions `raw` in `data.c` and comment it out and it'll probably work.
